test/e2e/tests/eg_upgrade.go

```diff
@@ -274,13 +275,14 @@ func migrateChartCRDs(actionConfig *action.Configuration, gatewayChart *chart.Ch
 	}
 
 	for _, crd := range crds {
-		if crd.Name == "backendtlspolicies.gateway.networking.k8s.io" {
+		if crd.Name == "backendtlspolicies.gateway.networking.k8s.io" ||
+			crd.Name == "grpcroutes.gateway.networking.k8s.io" {
```

Comment:
https://gateway-api.sigs.k8s.io/guides/?h=upgrade#v11-upgrade-notes

GRPCRoute graduated to GA.